Applications/SimpleShot/task_runner.cpp

```diff
@@ -3,10 +3,11 @@
  * Copyright (C) 2020 Jihoon Lee <jhoon.it.lee@samsung.com>
  *
  * @file   task_runner.cpp
- * @date   08 Jan 2021
- * @brief  task runner for the simpleshot demonstration
+ * @date   23 Sep 2021
+ * @brief  task runner for adding new classes at runtime for simpleshot learning
+ * demonstration
  * @see    https://github.com/nnstreamer/nntrainer
- * @author Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @author Priyank Mohan Verma <p1.verma@samsung.com>
```

Comment:
Add yourself as a new author with the existing authors like below:
```
 * @author Jihoon Lee <jhoon.it.lee@samsung.com>
 * @author Priyank Mohan Verma <p1.verma@samsung.com>
```

Reply:
Okay

Reply:
Please Check now